code/modules/clothing/under/miscellaneous.dm

```diff
@@ -342,15 +342,15 @@
 	worn_state = "crdress"
 
 /obj/item/clothing/under/hosformalmale
-	name = "head of security's formal uniform"
-	desc = "A male head of security's formal-wear, for special occasions."
+	name = "chief of security's formal uniform"
+	desc = "A male Chief of Security's formal-wear, for special occasions."
```

Comment:
```suggestion
	desc = "A male chief of security's formal-wear, for special occasions."
```

Reply:
There is no consistency. Descriptions are capitalized, it's a job title.